ical_test.go

```diff
@@ -284,3 +285,39 @@ func TestSetDate(b *testing.T) {
 		b.Run(tCase.Alias, testFn)
 	}
 }
+
+func TestRoundtripURI(b *testing.T) {
+	testCases := []struct {
+		Alias    string
+		Expected string
+	}{
+		{
+			Alias:    "empty_url",
+			Expected: "",
+		},
+		{
+			Alias:    "scheme_and_port",
+			Expected: "https://google.com:8080",
+		},
+	}
+
+	//nolint:scopelint
```

Comment:
LGTM, can we just get rid of this `nolint` comment? I don't really like cluttering the code with these.